gems/pending/fs/modules/LinuxMountProbe.rb

```diff
@@ -7,7 +7,7 @@ module LinuxMountProbe
 
   def self.probe(fs)
     unless LINUX_FS_TYPES.include?(fs.fsType)
-      $log.debug "LinuxMountProbe << FALSE because file system (#{fs.fsType}) is not supported" if $log
+      $log.error "LinuxMountProbe << FALSE because file system (#{fs.fsType}) is not supported" if $log
```

Comment:
I'm not sure this should be an error. Maybe warn or info would be a better choice.